pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -588,8 +609,15 @@ func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg
 	})
 	if err != nil {
 		klog.ErrorS(err, "Eviction manager: pod failed to evict", "pod", klog.KObj(pod))
+		otelSpan.AddEvent("pod failed to evict", trace.WithAttributes(
```

Comment:
Can we use RecordError instead?
```suggestion
		otelSpan.RecordError(fmt.Errorf("pod failed to evict: %w", err), trace.WithAttributes(
```